### PR TITLE
xlockmore: add missing license

### PR DESCRIPTION
**Pull request by cr6git**
Added license from the Arch [package](https://git.archlinux.org/svntogit/community.git/tree/trunk?h=packages/xlockmore).

### CODE REVIEW

**Comment by the-maldridge** [comment]
Please refactor this to obtain the license from an official source.  Versioning the license with void-packages will undoubtedly lead to an out of date license.

**Comment by Vaelatern** [comment]
It looks like it comes from a source file, so maybe judicious `sed` can get the license out for you.
See the spim template, line 18, for an example.

**Comment by cr6git** [comment]
@Vaelatern OK, I used `sed` to get the license text from `xlock/xlock.h`

File: srcpkgs/xlockmore/template [hunk 1]
```diff
@@ -21,4 +21,7 @@ do_install() {
 		bindir=${DESTDIR}/usr/bin mandir=${DESTDIR}/usr/share/man/man1 \
 		prefix=${DESTDIR}/usr install
 	chmod u+s ${DESTDIR}/usr/bin/xlock
+
+	sed -n 56,74p xlock/xlock.h > LICENSE
```
Comment (the-maldridge) [inline srcpkgs/xlockmore/template]: this looks really fragile.  Is it possible to do this with anchors in the text?

Comment (cr6git) [reply srcpkgs/xlockmore/template]: anchors ? excuse-me, what is it ?

Comment (the-maldridge) [reply srcpkgs/xlockmore/template]: like clipping from a point in the text by the start of license to the end of the license.  Though I doubt xlockmore chances much anymore.  If that is infeasible I am satisfied to merge it as is.

Comment (cr6git) [reply srcpkgs/xlockmore/template]: me I go to sleep.

**Pull request closed without merging**